sessionctx/variable/session.go

```diff
@@ -1134,29 +1135,31 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 	case TiDBOptConcurrencyFactor:
 		s.ConcurrencyFactor = tidbOptFloat64(val, DefOptConcurrencyFactor)
 	case TiDBIndexLookupConcurrency:
-		s.IndexLookupConcurrency = tidbOptPositiveInt32(val, DefIndexLookupConcurrency)
+		s.indexLookupConcurrency = tidbOptPositiveInt32Ptr(val)
```

Comment:
Better to append a warning message like https://github.com/pingcap/tidb/blob/622d40b5f28afd92a6d317fb8daf714afbdf545f/sessionctx/variable/session.go#L1187

Reply:
done